tox.ini

```diff
@@ -90,8 +92,10 @@ description = Lint the source code
 base = testenv, lint
 labels = lint
 allowlist_externals =
-    shellcheck: bash, xargs
+    bash
+    shellcheck: xargs
 commands_pre =
+    bash -c "[ -n \"$CI\" ] && echo \"::group::$TOX_ENV_NAME\" || exit 0"
```

Comment:
I would have liked to add `commands_pre` and `commands_post` once to the `[lint]` section on line 79, but they don't execute if `commands_pre` or `commands_post` are defined in the "child" sections (like this one).